recipe-server/normandy/base/api/serializers.py

```diff
@@ -17,3 +17,15 @@ class Meta:
             'last_name',
             'email',
         ]
+
+
+class ServiceInfoSerializer(serializers.ModelSerializer):
+    user = UserSerializer()
+    peer_approval_enforced = serializers.BooleanField()
+
+    class Meta:
```

Comment:
Oooops. I copy/pasted this from the UserSerializer above, which is where I got this.